src/test/java/it/gov/pagopa/pu/registry/mapper/debtposition/DebtPositionRegistryMapperServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.pu.registry.mapper.debtposition;
+
+import it.gov.pagopa.pu.registry.event.payments.dto.*;
+import it.gov.pagopa.pu.workflowhub.dto.generated.DebtPositionDTO;
+import it.gov.pagopa.pu.workflowhub.dto.generated.PaymentEventType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class DebtPositionRegistryMapperServiceTest {
+
+  private DebtPositionRegistryMapperService debtPositionRegistryMapperService;
+  private DebtPositionEventDTO2DebtPositionRegistryMapper debtPositionEventDTO2DebtPositionRegistryMapper;
+  private DebtPositionIoEventDTO2DebtPositionRegistryMapper debtPositionIoEventDTO2DebtPositionRegistryMapper;
+  private DebtPositionSendEventDTO2DebtPositionRegistryMapper debtPositionSendEventDTO2DebtPositionRegistryMapper;
+
+  @BeforeEach
+  void setUp() {
+    this.debtPositionEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionEventDTO2DebtPositionRegistryMapper());
+    this.debtPositionIoEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionIoEventDTO2DebtPositionRegistryMapper());
+    this.debtPositionSendEventDTO2DebtPositionRegistryMapper = Mockito.spy(new DebtPositionSendEventDTO2DebtPositionRegistryMapper());
```

Comment:
Instead of this, you could also do:
```suggestion
@ExtendWith(Mockito.extension)
class DebtPositionRegistryMapperServiceTest {

  @Spy
  private DebtPositionEventDTO2DebtPositionRegistryMapper debtPositionEventDTO2DebtPositionRegistryMapperSpy;
  @Spy
  private DebtPositionIoEventDTO2DebtPositionRegistryMapper debtPositionIoEventDTO2DebtPositionRegistryMapperSpy;
  @Spy
  private DebtPositionSendEventDTO2DebtPositionRegistryMapper debtPositionSendEventDTO2DebtPositionRegistryMapperSpy;
  
  private DebtPositionRegistryMapperService debtPositionRegistryMapperService;

  @BeforeEach
  void setUp() {
```

Reply:
ps: let's always add suffix `Spy` or `Mock` to immediately know if a service is a real one or not

Reply:
ps: why are you using spy and not mocks?
with a mock, the test could simpler

Reply:
I used Spy because I wanted to have the real underlying service structure, switched with Mock.